### PR TITLE
MypyBear: Use ``language`` correctly

### PR DESCRIPTION
**Pull request by adhikasp**
check done:
- [x] running pytest locally

Fixes https://github.com/coala/coala-bears/issues/2022

### CODE REVIEW

File: tests/python/MypyBearTest.py [hunk 1]
```diff
@@ -50,7 +52,8 @@ def test_py2_version(self):
                             ['print 123'])
 
     def test_bad_language(self):
```
Comment (jayvdb) [inline tests/python/MypyBearTest.py]: You can simply remove this test ; bad languages are not a bear problem now.

Comment (adhikasp) [reply tests/python/MypyBearTest.py]: Hm... actually this also test if `language` is not supported by the bear (required for 100% cov too)

**Comment by jayvdb** [comment]
ack 08cb3b1 52d75b5 2bff583

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**